nusantara/nusa_datasets/parallel_su_id/parallel_su_id.py

```diff
@@ -0,0 +1,119 @@
+from pathlib import Path
+from typing import List
+
+import datasets
+import json
+
+from nusantara.utils import schemas
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks, DEFAULT_SOURCE_VIEW_NAME, DEFAULT_NUSANTARA_VIEW_NAME
+
+_DATASETNAME = "parallel_su_id"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_NUSANTARA_VIEW_NAME
+
+_LANGUAGES = ["ind", "sun"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_LOCAL = False
+_CITATION = """\
+@INPROCEEDINGS{7437678,
+  author={Suryani, Arie Ardiyanti and Widyantoro, Dwi Hendratmo and Purwarianti, Ayu and Sudaryat, Yayat},
+  booktitle={2015 International Conference on Information Technology Systems and Innovation (ICITSI)}, 
+  title={Experiment on a phrase-based statistical machine translation using PoS Tag information for Sundanese into Indonesian}, 
+  year={2015},
+  volume={},
+  number={},
+  pages={1-6},
+  doi={10.1109/ICITSI.2015.7437678}}
+"""
+
+_DESCRIPTION = """\
+This data contains 3616 lines of Sundanese sentences taken from the online Sundanese language magazine Mangle, West Java Dakwah Council, and Balebat, and translated into Indonesian by several students of the Sundanese language study program UPI Bandung.
+"""
+
+_HOMEPAGE = "https://dataverse.telkomuniversity.ac.id/dataset.xhtml?persistentId=doi:10.34820/FK2/HDYWXW"
+
+_LICENSE = "Creative Commons CC0 - No Rights Reserved"
+
+_URLs = {"ind": "https://dataverse.telkomuniversity.ac.id/api/access/datafile/:persistentId?persistentId=doi:10.34820/FK2/HDYWXW/032QZD",
```

Comment:
The two links for Indonesian and Sundanese are the same, I think these two should be different

Reply:
addressed in the bugfix commit. Thanks!